.github/workflows/build-ubuntu.yml

```diff
@@ -24,19 +24,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04]
-        client-gg: ['ON', 'OFF']
-        client-godot: ['OFF']
+        os: [ubuntu-20.04]
+        client-gg: ['ON']
```

Comment:
Looks like client-gg is always ON.  Should we just remove it from the matrix?

Reply:
We disable it when test for `clang` but it could be tested too.